src/core/annotation.js

```diff
@@ -1201,7 +1195,7 @@ class WidgetAnnotation extends Annotation {
   getOperatorList(evaluator, task, renderForms, annotationStorage) {
     // Do not render form elements on the canvas when interactive forms are
     // enabled. The display layer is responsible for rendering them instead.
-    if (renderForms) {
+    if (!this._isSignature && renderForms) {
```

Comment:
Do we really need a boolean for this, i.e. can't we remove that property and instead do the following (to avoid having to track the kind twice)?
```suggestion
    if (renderForms && !(this instanceof SignatureWidgetAnnotation)) {
```